lib/src/textArea/textArea.component.scss

```diff
@@ -0,0 +1,71 @@
+@import "~@sebgroup/bootstrap/scss/variables";
+
+$transition-time: 200ms;
+
+.form-group.text-area {
+    > .input-field {
+        > textarea.form-control {
+            color: $body-color;
+            border-color: $gray-500;
+            transition: border $transition-time / 2;
+
+            &:not(.resizable) {
+                resize: none;
+            }
+
+            &:not(:disabled) {
+                &:hover {
+                    border-color: $blue;
+                    background-color: $white;
+                }
+
+                &:focus {
+                    border-color: $blue-darker;
+                    box-shadow: none;
+                    background-color: $white;
+
+                    &:active {
+                        border-color: $blue-darker;
+                        box-shadow: none;
+                        background-color: $white;
+                    }
+                }
+            }
+        }
+
+        > .alert {
+            opacity: 0;
+            margin: 0;
+            padding: 0;
+            background: transparent;
+            max-height: 0;
+            border: none;
+            border-radius: 0px;
+            overflow: hidden;
+            width: 100%;
+            transition: opacity $transition-time, padding $transition-time, max-height $transition-time * 2;
+
+            &.alert-danger {
+                color: $red;
+            }
+        }
```

Comment:
We can upgrade them later when we implement the said suggestion in react component. For now i suggest we have this reflect exactly what we have in react components.

Reply:
I don't think you need it. You already have an `ngIf` on the error message which will eliminate the transition

Reply:
That part i realised and recognised the fact that we don't have the if check in react component just because we want to retain the transition. However i don't feel like this is necessary. I rather we don't render the error section when not available considering we might have multiple textAreas in a page. What do you guys think ?

Reply:
I agree with Mario we shouldn't override alert since its used in most projects, please try to remove them.
Also Yousif is right, ngIf will eliminate the transition

Reply:
You're still overriding `alert` and `alert-danger`. If you insist on doing this at least do `alert custom-alert` and `alert-danger custom-alert-danger`

Reply:
@mario-subo we are going to change it. But starting with react-component. We agree to stick to boostrap classes . The reason why we can't change it here is because we want the component to mirror whatever we have right now in react-component for easy tracking. I will update it as soon as we update react-component.